src/components/common-components/autocomplete/autocomplete.component.tsx

```diff
@@ -64,32 +91,35 @@ export function AutocompleteComponent<T>({
         />
       </div>
       {groupedOptions.length > 0 ? (
-        <ul
+        <div
           ref={tooltipRef}
           className={classNames("listbox")}
           style={styles.popper}
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           onMouseDown={(getListboxProps() as any).onMouseDown}
         >
-          {groupedOptions.map((option, index) => (
-            <li
-              {...getOptionProps({ option, index })}
-              data-cy={option["data-cy"]}
-              onClick={(e: React.MouseEvent): void => {
-                e.stopPropagation();
-                setValue(option);
-              }}
-            >
-              <div className="container">
-                <div className="optionLabel">{getOptionLabel(option)}</div>
-                <div
-                  className="colorSample"
-                  style={{ backgroundColor: `#${getOptionColor(option)}` }}
-                />
-              </div>
-            </li>
-          ))}
-        </ul>
+          {groupedOptions.map((option, index) => {
+            if (option.name.trim() === "wolne") {
+              return <LabelComponent option={option} index={index} />;
+            }
+            return null;
+          })}
+          {groupedOptions.map((option, index) => {
+            if (option.name.trim() !== "wolne") {
+              if (option.from !== 0 && option.to !== 24)
+                return <LabelComponent option={option} index={index} />;
+            }
+            return null;
+          })}
+          <div className="autoSeparator" />
+          {groupedOptions.map((option, index) => {
+            if (option.name.trim() !== "wolne") {
+              if (option.from === 0 && option.to === 24)
```

Comment:
Czy dobrze rozumiem, że ty w tym miescu chcesz sprawdzić czy to jest zmiana niepracuąca?

Reply:
Tak

Reply:
Mamy pole `isWorkingShift` na modele więc proponuję użyć go, ponieważ jest bardziej deskryptywne. 
Oprócz tego to ma taką zaletę, że nie jednoznacznie opisuje zmiany niepracujące.
Bo w sumie nic nie przeszkadza żeby one wprowadziły jakąś dobę która przypadnie na te godziny

Czy ty miałeś jakąś motywację żeby tego nie robić?
![image](https://user-images.githubusercontent.com/21079319/111543075-d71dad80-8772-11eb-9b77-f4a568644c97.png)